home-manager/home-manager

```diff
@@ -183,7 +183,8 @@ function doBuildAttr() {
         "${extraArgs[@]}" \
         "${PASSTHROUGH_OPTS[@]}" \
         --argstr confPath "$HOME_MANAGER_CONFIG" \
-        --argstr confAttr "$HOME_MANAGER_CONFIG_ATTRIBUTE"
+        --argstr confAttr "$HOME_MANAGER_CONFIG_ATTRIBUTE" \
+      || exit $?
```

Comment:
I'd prefer to keep the old behavior, the error code should be handled by the caller.